PROJECT

```diff
@@ -8,7 +8,7 @@ layout:
 plugins:
   manifests.sdk.operatorframework.io/v2: {}
   scorecard.sdk.operatorframework.io/v2: {}
-projectName: openshift-lightspeed-operator
+projectName: lightspeed-operator
```

Comment:
AFAIR Gaurav mentioned 
`The approved short name (after first use of the full name) is "OpenShift Lightspeed"`
I understand why you made this change, just wondering if we should remove the word "openshift"

Reply:
This name is used CSV version , so blank space cannot be used. Also , it looked like openshift-lightspeed-operator is a long name , and I dont see other operators using prefix `openshift-`